data-raw/accessibility_2000.R

```diff
@@ -0,0 +1,30 @@
+library(sf)
+library(terra)
+library(mapme.biodiversity)
+
+# Load the spatial object (x) for cropping
+x <- read_sf(system.file("extdata", "sierra_de_neiba_478140.gpkg", package = "mapme.biodiversity"))
+
+# Set output directory for cropped data
+outdir <- "inst/resources/accessibility_2000"
```

Comment:
This should be `inst/res/accessibility_2000`. I'd guess you were basing your code on an artifact from the transfer of `mapme.indicators`. Sorry for that!